test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/AntiforgerySampleTest.cs

```diff
@@ -26,11 +27,19 @@ public AntiforgerySampleTests(AntiForgerySampleTestFixture fixture)
             var response = await Client.GetAsync("http://localhost/Index.html");
 
             // Assert
-            var cookie = RetrieveAntiforgeryCookie(response);
-            Assert.NotNull(cookie.Value);
-
-            var token = RetrieveAntiforgeryToken(response);
-            Assert.NotNull(token.Value);
+            var setCookieHeaderValue = RetrieveAntiforgeryCookie(response);
+            Assert.NotNull(setCookieHeaderValue);
+            Assert.False(string.IsNullOrEmpty(setCookieHeaderValue.Value));
+            Assert.Null(setCookieHeaderValue.Domain);
+            Assert.Equal("/", setCookieHeaderValue.Path);
+            Assert.False(setCookieHeaderValue.Secure);
+
+            setCookieHeaderValue = RetrieveAntiforgeryToken(response);
```

Comment:
Yes, I thought of moving to a common method, but I think in general we want to avoid these common methods as it immediately does not reflect the line where the test fails. I am not too strong about it though.